docs/make.jl

```diff
@@ -13,4 +15,11 @@ Downloads.download(
 )
 include("build_docs_with_style.jl")
 
-build_docs_with_style(pages, PeriodicOrbits)
\ No newline at end of file
+using DocumenterCitations
+
+bib = CitationBibliography(
+    joinpath(@__DIR__, "refs.bib");
+    style=:authoryear
+)
+
+build_docs_with_style(pages, PeriodicOrbits;bib)
```

Comment:
```suggestion
build_docs_with_style(pages, PeriodicOrbits; bib)
```

formatting again. I will set up auto formatter.